### ISSUE TITLE
Unable to open Joplin 1.0.220 on Macbook

### ISSUE THREAD

**Issue by quarkpt** [laurent22/joplin#3400]
## Environment

Joplin version: 1.0.220
Platform: Mac
OS specifics: OS X 10.15.5.

## Steps to reproduce

1. Install 1.0.220 on Macbook by downloading dmg file.
2. Attempt to open Joplin. 

## Describe what you expected to happen

Joplin opens.

**Comment by laurent22** [comment]
You have a Logfile section in your post but it's empty. In case it's not obvious we need that to make sense of your bug, otherwise we can only reply "works for me" and close the bug.

**Comment by quarkpt** [comment]
Unfortunately, the bug is that Joplin doesn't start at all. I'm looking at log.txt in ~/.config/joplin-desktop, and the last entry is from June 21 at 10:41 AM, even though I attempted to launch Joplin about 5 minutes ago. None of my other failed attempts to launch it produced any logging output.

I also attempted to follow the instructions on

https://joplinapp.org/debugging/#desktop-application

but -- again -- we didn't get far enough for any logging output to get produced.
